cmd/linux.go

```diff
@@ -115,8 +114,10 @@ func parseOsRelease(osRelease string) *DistroInfo {
 		result.Distribution = Unknown
 	}
 
-	result.ID = osID
 	result.Name = osNAME
+	result.ID = osID
+	result.Release = version
```

Comment:
changes here are just an attempt to clean up the code and make it more obvious what is going on. I didn't commented, as the linux part needs more comments